legion/engine/physics/colliders/convexcollider.cpp

```diff
@@ -73,15 +94,19 @@ namespace legion::physics
 
         std::array<std::shared_ptr<PenetrationQuery>, 3> penetrationQueryArray{ abEdgePenetrationQuery, abPenetrationQuery, baPenetrationQuery  };
 
-        auto lessThan
-        {
-            []( std::shared_ptr<PenetrationQuery> lhs,  std::shared_ptr<PenetrationQuery> rhs)
-            {
-            return lhs->penetration < rhs->penetration;
-            }
-        };
+        /*log::debug("abPenetrationQuery with normal {} , has penetration {}", abPenetrationQuery->normal,abPenetrationQuery->penetration);
+        log::debug("baPenetrationQuery n with normal {} ,  has penetration {}", baPenetrationQuery->normal,baPenetrationQuery->penetration);
+        log::debug("abEdgePenetrationQuery with normal {} ,  has penetration {}", abEdgePenetrationQuery->normal ,abEdgePenetrationQuery->penetration);*/
+
+        //auto lessThan
+        //{
+        //    []( std::shared_ptr<PenetrationQuery> lhs,  std::shared_ptr<PenetrationQuery> rhs)
+        //    {
+        //    return lhs->penetration < rhs->penetration;
+        //    }
+        //};
 
-        //manifold.penetrationInformation = *std::max_element(penetrationQueryArray.begin(), penetrationQueryArray.end(),lessThan);
+        ////manifold.penetrationInformation = *std::max_element(penetrationQueryArray.begin(), penetrationQueryArray.end(),lessThan);
```

Comment:
if even the out-commented code has been commented out (double comment) it can probably be removed :)